src/BenchmarksApps/TodosApi/TodosApi.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <ServerGarbageCollection>false</ServerGarbageCollection>
```

Comment:
Any particular reason server garbage collection is disabled?

Reply:
These projects associated with our native AOT work are using Workstation GC until some on-going work in Server GC is comlpeted later in .NET 8

Reply:
I see. Is it possible to find an issue about that on-going work?

Reply:
dotnet/runtime#79658

Reply:
That issue doesn't have that good of a description, but it seems to be about improving GC (adjust heap count on load, better handle burst load) and has nothing to do with native aot. So I fail to see why we're not going to use Server GC.

Reply:
I'm afraid I still don't follow. Since ideally we want to test with server GC only (as it's intended for server apps), to me it makes sense to already start testing with it, so we can notice any possible regressions. A good example here is npgsql, which isn't trimming safe yet, but we still use the latest preview.

Reply:
We do test with server GC as well. That's defined in the `goldilocks.benchmarks.yml` file.

Reply:
You probably meant `nativeaot-scenarios.yml` which indeed enables server GC. Which makes even less sense as to why disable it here. Just so people running it locally will see pretty numbers?